charts/rasa/Chart.yaml

```
@@ -13,12 +13,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.16.2
+version: 1.16.3
```

```suggestion
version: 1.17.0
```
If the app version is bumped, then it should be minor bumped.